src/main.c

```diff
@@ -38,12 +39,38 @@ void SysTick_Handler(void) {
   msTicks++;
 }
 
+void TIMER32_0_IRQHandler(void) {
+	Chip_TIMER_Reset(LPC_TIMER32_0);		        /* Reset the timer immediately */
+	Chip_TIMER_ClearCapture(LPC_TIMER32_0, 0);	    /* Clear the capture */
+	adc_input.wheel_1_clock_cycles_between_ticks = 
+        Chip_TIMER_ReadCapture(LPC_TIMER32_0, 0);
+}
+
+void TIMER32_1_IRQHandler(void) {
+	Chip_TIMER_Reset(LPC_TIMER32_1);		        /* Reset the timer immediately */
+	Chip_TIMER_ClearCapture(LPC_TIMER32_1, 0);	    /* Clear the capture */
+	adc_input.wheel_2_clock_cycles_between_ticks = 
+        Chip_TIMER_ReadCapture(LPC_TIMER32_1, 0);
+}
+
+void Set_Interrupt_Priorities(void) {
+	/* Give 32 bit timer capture interrupts the highest priority */
+    NVIC_SetPriority(TIMER_32_0_IRQn, 0);
+    NVIC_SetPriority(TIMER_32_1_IRQn, 1);
+    /* Give the SysTick function a lower priority */
+	NVIC_SetPriority(SysTick_IRQn, 2);	
+}
+
 void Init_ADC_Structs(void) {
   adc_input.steering_raw = 0;
   adc_input.accel_1_raw = 0;
   adc_input.accel_2_raw = 0;
   adc_input.brake_1_raw = 0;
   adc_input.brake_2_raw = 0;
+  adc_input.wheel_1_clock_cycles_between_ticks = 0; 
+  adc_input.wheel_2_clock_cycles_between_ticks = 0;
+  // TODO: should these be initialized to zero? Zero clock cycles between ticks would
```

Comment:
ya, just make sure to check whenever you use them anywhere that they are not zero, otherwise it means they haven't been initialized yet (and you will get a divide by zero error and mega crash). seems fine.